container/build.py

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/env python
+
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Builds the fully-loaded container, with Google Cloud Container Builder or locally.
+
+To build with Google Cloud Container Builder:
+$ python build.py -p my-gcp-project -d {container_type} -c {container_name} -t latest -b my_bucket
+will produce docker images in Google Container Registry:
+    gcr.io/my-gcp-project/{container_type}:latest
+and the debian packages installed will be packed as a tarball and stored in
+gs://my_bucket for future reference, if -b is specified.
+To build locally:
+$ python build.py -d {container_type} -l
+will produce docker locally as {container_type}:latest
+
+usage:
+  build.py -d,
```

Comment:
is the ',' supposed to be there after the -d?

Reply:
That's how python generated the help text, but I changed it to look a bit better